src/components/vendor-links/vendor-links.tsx

```diff
@@ -49,6 +49,10 @@ export const OTALink: React.FunctionComponent<VendorProps> = (props: VendorProps
             url = `https://ww8.ikea.com/ikeahomesmart/releasenotes/releasenotes.html`
             break
 
+        case "Philips":
+            url = `https://www.philips-hue.com/en-us/support/release-notes`
```

Comment:
Can we try to guess what type of device is this using `device.definition` if yes, we can land users to either
`https://www.philips-hue.com/en-us/support/release-notes/accessories` or `https://www.philips-hue.com/en-us/support/release-notes/lamps`

Reply:
@nurikk Great suggestion - I've done this by checking `device.definition?.exposes` for a `light` capability and linking to lamps if found, and to accessories for everything else.